ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueCommitting.scala

```diff
@@ -34,10 +34,13 @@ import scala.collection.JavaConverters._
 class KeyValueCommitting private[daml] (
     engine: Engine,
     metrics: Metrics,
-    inStaticTimeMode: Boolean) {
+    inStaticTimeMode: Boolean,
+    packageValidation: CommitPackageValidationMode,
+) {
   private val logger = LoggerFactory.getLogger(this.getClass)
 
-  def this(engine: Engine, metrics: Metrics) = this(engine, metrics, false)
+  def this(engine: Engine, metrics: Metrics) =
+    this(engine, metrics, false, CommitPackageValidationMode.PostCommit)
```

Comment:
We should avoid having multiple places with default values. I think this constructor should also have the `packageValidation` parameter. Any concerns with such an API change, @miklos-da?

Reply:
I dropped the default values.